packages/panamax/scripts/trim-index.py

```
@@ -0,0 +1,196 @@
+        if removed:
+            modified_files += 1
+            rel = os.path.relpath(abs_path, mirrors_dir)
+            if deleted:
+                deleted_files += 1
+                print(f"  {rel}: deleted (all {removed} version(s) missing)")
+            else:
+                print(f"  {rel}: kept {kept}, removed {removed} version(s)")
+
+    # Remove empty directories bottom-up (leaves first)
+    deleted_dirs = 0
+    for root, dirs, files in os.walk(index_dir, topdown=False):
```

Ruff (0.15.12)

[warning] Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)